component-api/src/main/java/org/talend/sdk/component/api/component/AfterVariableContainer.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2006-2021 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.api.component;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Mark method that return container with after variables.
+ */
+@Target(ElementType.METHOD)
+@Retention(RUNTIME)
+public @interface AfterVariableContainer {
```

Comment:
Can't we link this notion to `@AfterVariables` annotation with a value name like `method` or smtg like this? So, we can get rid of this annotation...
Retain this comment on hold, coz reading the rest of code...

Reply:
I'm gonna move this annotation to AfterVariables as an inner class.
I think to keep it. This annotation allows us to change implementation easier if we want to in the future. This annotation just marks something as the container of after variables. And the container may change. For example, we can mark a field, instead of a method, or even use some kind of @Service with it. 
Also, the connector developer isn't forced to use @AfterVariables annotation. It's optional because users can use repeatable @AfterVariable annotations instead.